src/seedu/addressbook/commands/Command.java

```diff
@@ -63,6 +63,11 @@ protected ReadOnlyPerson getTargetPerson() throws IndexOutOfBoundsException {
         return relevantPersons.get(getTargetIndex() - DISPLAYED_INDEX_OFFSET);
     }
 
+    /**
+     * Checks if the data file is changed after entering the command
```

Comment:
Try to use the phrasing "return true if ...".

Reply:
Done!